packages/core-mobile/app/store/posthog/types.ts

```diff
@@ -27,7 +27,12 @@ export const DefaultFeatureFlagConfig = {
   [FeatureGates.SEEDLESS_SIGNING]: true,
   [FeatureGates.BLOCKAID_TRANSACTION_VALIDATION]: true,
   [FeatureGates.BLOCKAID_DAPP_SCAN]: true,
-  [FeatureGates.BALANCE_CHANGE_NOTIFICATIONS]: false
+  [FeatureGates.BALANCE_CHANGE_NOTIFICATIONS]: false,
```

Comment:
let's change BALANCE_CHANGE_NOTIFICATIONS to true by default as well

Reply:
fixed.